caddyhttp/httpserver/plugin.go

```diff
@@ -447,6 +447,8 @@ var directives = []string{
 	"realip", // github.com/captncraig/caddy-realip
 	"git",    // github.com/abiosoft/caddy-git
 
+	"cache",
```

Comment:
After looking the order for a while I cannot find the correct place. I think it should be after `log` and `errors` but it also should be before `gzip` and maybe even before `ratelimit`, but `errors` is after `gzip`. 

Although being after `gzip` (or `minify`) will work, caching gzipped responses can save a lot of cpu. As cache goes lower in the list, more duplicated tasks will be made, defeating the cache purpose which is avoid them. Maybe moving errors after log and putting cache after (log -> error -> cache) would solve the issue.

Being before rewrite should not be a problem because given an request url the rewrite should be the same and caching still applies.